AdobeDocs/commerce-webapi
# Update catalog-pricing.md

Added a note for the special price endpoint for Adobe Commerce users. Price_from and price_to are mandatory parameters that triggers a catalog schedule. Having a price scope set on "website", the possibility a store_id can be given and assuming you can use overlapping date-time ranges for price_from and price_to will trigger the error: "Future Update already exists in this time range. Set a different range and try again."



## Description



## Related Issue






## Motivation and Context



## How Has This Been Tested?





1. Setup a multi store webshop in Adobe Commerce 
2. Set the price scope on "Website" (Stores > Settings > Configuration > Catalog > Catalog > Price > Catalog Price Scope)
3. Do a POST API-call to: <host>/rest/<scope or all>/V1/products/special-price with the following body: { "prices": [ { "price": 33.33, "store_id": <store_id> (1), "sku": "<sku> (123)", "price_from": "2023-03-09 08:00:00", "price_to": "2023-03-09 23:59:00" } ] } 

Result: [] 

4. Do another POST API-call to: <host>/rest/<scope or all>/V1/products/special-price { "prices": [ { "price": 33.33, "store_id": <store_id> (2), "sku": "<sku> (123)", "price_from": "2023-03-09 12:00:00", "price_to": "2023-03-09 23:59:00" } ] } 

Expected result: []
Actual result: [ { "message": "Future Update already exists in this time range. Set a different range and try again.", "parameters": [ "33.33", "<sku> (123)", "<store_id> (2),", "price_from": "2023-03-09 12:00:00", "price_to": "2023-03-09 23:59:00" ] } ]

## Screenshots (if appropriate):

## Types of changes




## Checklist:
